pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,332 @@
+            </a>
+          </div>
+          <img
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/dua_wide-41767488aed1f75828f980b6b41339a0c65d4964.jpg?s=600"
+          >
+        </article>
+        <article class="article-slide">
+          <div class="article-description">
+            <a href="https://www.npr.org/sections/national/">
+              <h2>National</h2>
+            </a>
+            <a
+              href="https://www.npr.org/2020/12/03/942034617/time-names-its-kid-of-the-year-water-testing-scientist-gitanjali-rao"
+            >
+              <amp-fit-text height="70" min-font-size="20" max-font-size="20">
+                'Time' Names Its Kid Of The Year: Water-Testing Scientist
+                Gitanjali Rao
+              </amp-fit-text>
+            </a>
+          </div>
+          <img
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/time201207_koty.coverfinal_wide-a7cff22dc41f55823d394817f2c1a0acfc105a66.jpg?s=600"
+            layout="intrinsic"
+          >
+        </article>
+        <article class="article-slide">
+          <div class="article-description">
+            <a href="https://www.npr.org/sections/book-reviews/">
+              <h2>Book Reviews</h2>
+            </a>
+            <a
+              href="https://www.npr.org/2020/12/04/941982629/maureen-corrigans-10-books-that-will-connect-you-in-a-socially-distant-year"
+            >
+              <amp-fit-text height="70" min-font-size="20" max-font-size="20">
+                Maureen Corrigan's 10 Books That Will Connect You In A Socially
+                Distant Year
+              </amp-fit-text>
+            </a>
+          </div>
+          <img
+            class="article-img"
+            width="700"
+            height="393"
+            src="https://media.npr.org/assets/img/2020/12/03/corrigan2020_wide-26e0b93fd657db854e1ba2bdb7b18539ca72ecfa-s1100-c85.jpg"
+          >
+        </article>
+      </amp-base-carousel>
+    </div>
+  </body>
+</html>
+```
+
+See the [demo in action on Glitch](TODO).
+
+# Managing events
+
+Fully valid AMP installs event listeners on elements via the [`on` attribute](../learn/amp-actions-and-events.md) with the event and responding action as values. Bento AMP does not rely on this attribute. Instead, the AMP team will provide a JS API to manage and react to events. 
+
+During the developer preview, you can register component events and respond with defined actions using following syntax:
+
+`el.getApi().then(api => api.play())`
+
+The code below triggers [`amp-base-carousel`'s next action](../../../documentation/components/reference/amp-base-carousel.md#next) when executed. 
+
+```js
+const carousel = $('amp--base-carousel');
+const apiPromise = carousel.getApi();
+apiPromise.then(api => {
+  api.next();
+})
+```
```

Comment:
Maybe add a sentence mentioning where component APIs are documented.

Reply:
> Left a few additional comments. We also need a section describing how layouts work with Bento.

I've ben thinking a bit about this, and talking about AMP layouts and reproducing them seems not useful to non-AMP developers, which I think is a target audience for this document? Mentioning the AMP layouts, and that they can't be used seems like it would be confusing for developers who have never used AMP. 

Is there something specific to these components around dimensions and layout that is standard across the components that isn't dependent on the tech stack the developer is inserting the components into?

Reply:
The target audience is AMP and non-AMP developers. Imo mentioning that AMP layouts are not supported is a must, but it does not need to be the first thing to mention. 

> Is there something specific to these components around dimensions and layout that is standard across the components that isn't dependent on the tech stack the developer is inserting the components into?

//cc @caroqliu @dvoytenko

Reply:
Are the API documented anywhere that I could link to? @caroqliu @dvoytenko

Reply:
Not yet. That's the same question as we are trying to figure out in the amp-social-share's PR. We need to come up with a structure here that works for us.